docs/static/ds4/docs.min.js

```diff
@@ -19,47 +19,53 @@ $_mod.def("/makeup-screenreader-trap$0.0.5/util",function(n,t,e,r,o){"use strict
 $_mod.def("/makeup-modal$0.0.4/index",function(e,t,n,a,r){"use strict";function d(){if(p){o.untrap(p),i.untrap(p);var e=document.createEvent("Event");e.initEvent("unmodal",!1,!0),p.dispatchEvent(e),p=null}return p}function u(e){d(),p=e,o.trap(p),i.trap(p);var t=document.createEvent("Event");return t.initEvent("modal",!1,!0),p.dispatchEvent(t),p}var i=e("/makeup-keyboard-trap$0.0.9/index"),o=e("/makeup-screenreader-trap$0.0.5/index"),p=void 0;n.exports={modal:u,unmodal:d}});
 $_mod.def("/@ebay/skin$7.0.3/docs/src/js/base/transition",function(e,n,t,i,r){function o(e){for(var n=window.getComputedStyle(e).transitionDuration.replace(s,"")?1:0,t=e.firstElementChild;t;)n+=o(t),t=t.nextElementSibling;return n}function a(e){var n,t;return window.requestAnimationFrame?(n=requestAnimationFrame(function(){n=requestAnimationFrame(e)}),t=cancelAnimationFrame):(n=setTimeout(e,26),t=clearTimeout),function(){n&&(t(n),n=void 0)}}var m="transitionend",s=/0m?s(?:, )?/g;t.exports=function(e,n,t){function i(){s||(s=!0,e.removeEventListener(m,r,!0),f?(f(),d.remove(v)):d.remove(n))}function r(){++c===u&&(s=!0,e.removeEventListener(m,r,!0),d.remove(n),t&&t())}var s,u,c=0,d=e.classList,v=n+"-init",f=a(function(){e.addEventListener(m,r,!0),d.add(n),d.remove(v),u=o(e),f=void 0,0===u&&i()});return d.add(v),i}});
-$_mod.installed("@ebay/skin$7.0.3","makeup-roving-tabindex","0.1.1");
-$_mod.main("/makeup-roving-tabindex$0.1.1","");
-$_mod.installed("makeup-roving-tabindex$0.1.1","makeup-navigation-emitter","0.1.2");
-$_mod.main("/makeup-navigation-emitter$0.1.2","");
-$_mod.installed("makeup-navigation-emitter$0.1.2","custom-event-polyfill","0.3.0");
-$_mod.def("/makeup-navigation-emitter$0.1.2/util",function(t,e,r,i,o){"use strict";function n(t){return Array.prototype.slice.call(t)}r.exports={nodeListToArray:n}});
-$_mod.installed("makeup-navigation-emitter$0.1.2","makeup-key-emitter","0.0.3");
+$_mod.installed("@ebay/skin$7.0.3","makeup-roving-tabindex","0.1.0");
+$_mod.main("/makeup-roving-tabindex$0.1.0","");
+$_mod.installed("makeup-roving-tabindex$0.1.0","makeup-navigation-emitter","0.1.0");
```

Comment:
@seangates We need to be careful and remember to run Yarn when switching branches. These makeup versions have gone back and forth a few times now (I'm guilty of it too!). Will sync up the master branch with latest.